lib/tool_shed/webapp/templates/webapps/tool_shed/base_panels.mako

```diff
@@ -138,17 +138,12 @@
                 ## Help tab.
                 <%
                     menu_options = []
-                    qa_url = app.config.get( "qa_url", None )
-                    if qa_url:
-                        menu_options = [ [_('Galaxy Q&A'), qa_url, "_blank" ] ]
                     menu_options.extend( [
                         [_('Tool Shed Wiki'), app.config.get( "wiki_url", "https://galaxyproject.org/toolshed" ), "_blank" ],
```

Comment:
Rename `Tool Shed Wiki` as well? Not sure how though!

Reply:
What do you think it should be renamed to? I don't even see this as a option in the Help Menu, in fact.

Reply:
This is part of the [ToolShed](https://toolshed.g2.bx.psu.edu/) help menu, not Galaxy. Maybe we can call this "About" ?